src/pages/contributors/sacha-labourey.adoc

```diff
@@ -0,0 +1,185 @@
+Right now, the wave is AI and if people are curious enough, I think they'll discover that the relationship between AI and Jenkins can be extremely symbiotic.
+There is so much to discover in AI, that I think it would be wrong to be too static in how you wire things.
+I actually think the Jenkins architecture and ecosystem are a perfect host to combine those concepts together.
+I see no reason why Jenkins would not still be going strong in 5 to 10 years, since there is still a lot to be done.
+
+== Do you view supporting open-source as part of a company's corporate social responsibility and what advice would you give other C-suite executives on the importance of supporting Open Source?
+
+I think it's like a cake where you have multiple layers.
+The first one for me that's the most important is just Open Source is Open Source, meaning you have a set of freedoms in Open Source that have been formalized by the OSI and that's it.
+If you want to just say one thing it would be this, no more no less.
+I'm repeating this because I think for a lot of people, that was lost in translation.
+We've seen open-core models and we've seen companies moving to fake open-source licenses.
+We've seen a lot of different moves, and that almost made people be way too pragmatic about it.
+The notion that "If everything was equal it's kind of like Open Source" is incorrect.
+Either it is Open Source or it's not.
+
+You can find dozens and dozens of open-source licenses, so it's not very restrictive, but you have a number of things that it enables that are absolutely critical and I always go back to this.
+Sometimes I hear people say "It's not really Open Source if you do this or that" and I just want to say, as long as you respect the OSI mandate of what is Open Source, in my book it's good.
+Maybe you don't like the way a project is managed.
+That's absolutely fair but if you want to do something differently, fork it.
+That's the freedom you have, so I think it all boils down to this.
+That's really the first layer for me.
+
+Obviously, there is a lot more that should be done by organizations, but I want to stress that this first layer is very important.
+Beyond that, companies don't need to necessarily support Open Source with cash or with other resources.
+I think there are multiple ways to provide support.
+What I typically advise companies is more generic advice which is to take care of the ecosystem in which you evolve.
+If you have an ecosystem, treat that ecosystem well because if you don't, you're going to suffer as a part of that ecosystem.
+Especially today, where most companies have Open Source at the core of their ecosystem.
+They should do the right thing for Open Source, in whatever shape or form it may take, and they should think about how it could affect their company down the line.
```

Comment:
```suggestion
They should do the right thing for open source, in whatever shape or form it may take, and they should think about how it could affect their company down the line.
```